### DISCUSSION TITLE
Add 2fa authentication

### DISCUSSION THREAD

**Discussion by Giuseppe-TD** [ZoeyVid/NPMplus#1010] [Q&A] [answered]
As the ttitle, is possible to add 2fa authentication? With totp

**Comment by Zoey2936** [comment, marked as answer]
With the php rewrite (the function is already written)

**Reply by Giuseppe-TD** [reply]
ok thanks

**Reply by latimeria68** [reply]
Hello,
can you share and example and procedure to have It?
Thank you
